.github/workflows/ci.yml

```diff
@@ -416,6 +416,8 @@ jobs:
       - name: Build Documentation
         uses: actions-rs/cargo@v1
         with:
+          env:
+            RUSTDOCFLAGS: -Dwarnings
```

Comment:
It looks like this change broke the CI script, it now fails to parse with the following error:

```
The workflow is not valid. .github/workflows/ci.yml (Line: 420, Col: 13): A mapping was not expected
```

Reply:
It shouldn't have been in the `with:` binding, and I hadn't seen that because the CI workflow requires approval to run (which is silly).